web-workers/simple-shared-worker/multiply.js

```diff
@@ -6,15 +6,12 @@ const result1 = document.querySelector(".result1");
 if (!!window.SharedWorker) {
   const myWorker = new SharedWorker("worker.js");
 
-  first.onchange = function () {
-    myWorker.port.postMessage([first.value, second.value]);
-    console.log("Message posted to worker");
-  };
-
-  second.onchange = function () {
-    myWorker.port.postMessage([first.value, second.value]);
-    console.log("Message posted to worker");
-  };
+  [first, second].forEach(input => {
+    input.onchange = function() {
+      myWorker.port.postMessage([first.value, second.value]);
+      console.log('Message posted to worker');
+    }
+  })
 
   myWorker.port.onmessage = function (e) {
```

Comment:
We can also do

```suggestion
  myWorker.port.onmessage = (e) => {
```

Reply:
In fact, this is already changed in the content repo, so let's make it consistent